lib/utils/remote-build.ts

```diff
@@ -142,6 +144,7 @@ export async function startRemoteBuild(build: RemoteBuild): Promise<void> {
 		globalLogger.outputDeferredMessages();
 		await cancellationPromise;
 	}
+	return build.releaseId;
```

Comment:
`handleRemoteBuildStream` captures the release ID, but `handleHeadlessBuildStream` currently does not. The latter needs to be updated to capture the release ID. Looks like just a matter of adding a `build` argument, then:
```
build.releaseId = message.releaseId;
```
(Please test it.) :-)